snmp/tests/test_e2e_core_vs_python.py

```diff
@@ -41,7 +44,9 @@
 
 def test_e2e_metric_types(dd_agent_check):
     instance = common.generate_container_instance_config(SUPPORTED_METRIC_TYPES)
-    assert_python_vs_core(dd_agent_check, instance, expected_total_count=10 + 5)
+    assert_python_vs_core(
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Maybe we can just add `SKIPPED_CORE_ONLY_METRICS` inside `assert_python_vs_core` to avoid having to pass it every time the function is called ? by appending to `metrics_to_skip`